examples/basic/annotations/slope.py

```diff
@@ -1,3 +1,11 @@
+''' A marker plot that demonstrates a slope.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.models.slope, bokeh.plotting.figure.circle
+    :refs: :ref:`ug_basic_annotations_slopes`
```

Comment:
the existing ref is not pluralized
```suggestion
    :refs: :ref:`ug_basic_annotations_slope`
```